include/simple2d.h

```diff
@@ -130,13 +135,16 @@ typedef struct {
   int direction;
   int axis;
   int value;
+  int64_t finger_id;
+  int64_t touch_id;
```

Comment:
Its not done for SDL_JoystickID either, but I don't think its a big deal except it leaks SDL into Simple2D more than is already done